website_multi_theme/models/website.py

```diff
@@ -48,6 +48,10 @@ def write(self, vals):
             self._multi_theme_activate()
         return result
 
+    def _find_duplicate_view_for_website(self, origin_view, website):
+        xmlid = VIEW_KEY % (website.id, origin_view.id)
+        return self.env.ref(xmlid, raise_if_not_found=False)
```

Comment:
Are you sure of this? Shouldn't you search the view by the combination of `key` and `website_id` in `ir.ui.view`?

I think this needs tests :thinking: